apicode/newapi.py

```
@@ -0,0 +1,100 @@
+import requests
+import inquirer
+import textwrap
+import os
+import texttable as tt
+from inquirer.themes import GreenPassion
+
+
+class NewsApi:
+    # function for picking the news source from the user
+    def get_user_source(self):
+        questions = [
+            inquirer.List('sources',
+                          message="From which news source would you like to acquire your headlines?",
+                          choices=['nbc-news', 'cnbc',
+                                   'bbc-news', 'cnn'],
+                          ),
+
+
+        ]
+        # theme adds a green background to the items in the list
+        answers = inquirer.prompt(questions, theme=GreenPassion())
+        print("News Source : ", answers['sources'])
+
+        return answers['sources']
+    # function to retrieve a product key for the environment variable
+
+    def get_api_key(self):
+        try:
+            os.environ.get('API_KEY')
+        except Exception:
+
+            if 'API_KEY' not in os.environ:
+                print(
```

Would be better to raise an exception, to show that the api key is missing